src/Core/Grid/Definition/Factory/FeatureGridDefinitionFactory.php

```diff
@@ -0,0 +1,260 @@
+//                                    'route' => 'admin_features_view',
+//                                    'route_param_name' => 'featureId',
+//                                    'route_param_field' => 'id_feature',
+//                                ])
+//                        )
+//                        ->add(
+//                            (new LinkRowAction('edit'))
+//                                ->setName($this->trans('Edit', [], 'Admin.Actions'))
+//                                ->setIcon('edit')
+//                                ->setOptions([
+//                                    'route' => 'admin_features_edit',
+//                                    'route_param_name' => 'featureId',
+//                                    'route_param_field' => 'id_feature',
+//                                ])
+//                        )
+                        ->add((new SubmitRowAction('delete'))
+                            ->setName($this->trans('Delete', [], 'Admin.Actions'))
+                            ->setIcon('delete')
+                            ->setOptions([
+                                'method' => 'POST',
+                                'route' => 'admin_features_delete',
+                                'route_param_name' => 'featureId',
+                                'route_param_field' => 'id_feature',
+                                'confirm_message' => $this->trans(
+                                    'Delete selected item?',
+                                    [],
+                                    'Admin.Notifications.Warning'
+                                ),
+                            ])
+                        ),
+                ])
+            );
+
+        return $columns;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters()
+    {
+        $filters = (new FilterCollection())
+            ->add(
+                (new Filter('id_feature', NumberType::class))
+                    ->setTypeOptions([
+                        'attr' => [
+                            'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                        ],
+                        'required' => false,
+                    ])
+                    ->setAssociatedColumn('id_feature')
+            )
+            ->add(
+                (new Filter('name', TextType::class))
+                    ->setTypeOptions([
+                        'attr' => [
+                            'placeholder' => $this->trans('Search name', [], 'Admin.Actions'),
+                        ],
+                        'required' => false,
+                    ])
+                    ->setAssociatedColumn('name')
+            )
+            ->add(
+                (new Filter('values_count', NumberType::class))
+                    ->setTypeOptions([
+                        'attr' => [
+                            'placeholder' => $this->trans('Search values', [], 'Admin.Actions'),
```

Comment:
What are we expecting for this search: the number of values or the name of a value? I'm wondering if it needs to remain in the plural form or if it could be in the singular. 🤔

Reply:
It allows searching the number of values :slightly_smiling_face:

Reply:
Alright, thank you @rokaszygmantas. In this case, I would be more specific and write _Search values number_ instead, I find it clearer.

Reply:
updated it